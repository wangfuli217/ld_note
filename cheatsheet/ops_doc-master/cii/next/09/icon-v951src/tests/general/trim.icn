#SRC: JCON

procedure main()
    write(image(trim("abc  ")))
    write(image(trim("abc  ", ' ')))
    write(image(trim("abc  ", 'x')))
    write(image(trim("abc  ", 'abc ')))
    write(image(trim("a c  ", ' ')))
    write(image(trim("a c", ' ')))
    write(image(trim("   ", ' ')))
    write(image(trim(" ", ' ')))
    write(image(trim("", ' ')))
    write(image(trim("")))
end
