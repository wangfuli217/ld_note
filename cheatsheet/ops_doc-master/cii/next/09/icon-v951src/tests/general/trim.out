"abc"
"abc"
"abc  "
""
"a c"
"a c"
""
""
""
""
